src/remote/watcher.coffee

```diff
@@ -77,8 +79,7 @@ class RemoteWatcher
                         @whenReady callback
             else
                 @changes = @couch.client.changes
-                    filter: (doc) ->
-                        doc.docType?.toLowerCase() in ['file', 'folder']
+                    filter: @filterName
```

Comment:
I think it should be:

```coffee
filter: '_view'
view: @filterName
```

By the way, it's one more reason to call it a view and not a filter :wink: